docs/variants/novacustom_nv4x_tgl/test-matrix.md

```diff
@@ -117,88 +136,10 @@ subjected from before the release of the new binary.
 | 1.   | [USB Type-A devices detection][SUD]               | SUD           | All                                  |
 | 2.   | [M.2 Wi-fi][SMW]                                  | SMW           | All                                  |
 | 3.   | [NVMe detection][SNV]                             | SNV           | All                                  |
+| 4.   | [NET interface detection][NET]                    | NET           | All                                  |
+| 5.   | [TPM detection][TPD]                              | TPD           | TPD003.001, TPD004.001               |
 
 [SUD]: ../../unified-test-documentation/dasharo-stability/C01-usb-type-a-devices-detection.md
 [SMW]: ../../unified-test-documentation/dasharo-stability/C02-m2-wi-fi.md
 [SNV]: ../../unified-test-documentation/dasharo-stability/C03-nvme-detection.md
-
-## NV41MZ test matrix
```

Comment:
Why do we completely remove NV41MZ?

Reply:
Had to do that by mistake, fixed in 53b374c64145ec09e899ea4854ae1874ca8a0473